app/controllers/bugs_controller.rb

```diff
@@ -0,0 +1,55 @@
+class BugsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :authenticate_control!
+
+  def index
+    @bugs = Bug.where(game: current_game).order(:created_at)
+  end
+
+  def new
+    @bug = Bug.new
+    @teams = Team.all
+  end
+
+  def create
+    targetId = bug_params['target'].to_i
+    beneficiaryId = bug_params['beneficiary'].to_i
+    new_params = bug_params.except('target', 'beneficiary')
+    @bug = Bug.new(new_params)
+    @bug.target = Team.find(targetId)
+    @bug.beneficiary = Team.find(beneficiaryId)
+    @bug.klass = Message
```

Comment:
The idea is that bugs could become polymorphic, handling all the activity in a single place. `klass` might not be the right name, but that's the idea